src/components/picker/helpers/usePickerMigrationWarnings.tsx

```diff
@@ -9,7 +9,7 @@ type UsePickerMigrationWarnings = Pick<
 >;
 
 const usePickerMigrationWarnings = (props: UsePickerMigrationWarnings) => {
-  const {children, /* migrate, */ getItemLabel, getItemValue, onShow} = props;
+  const {children, getItemLabel, getItemValue, onShow} = props;
```

Comment:
Let's again add migrate here.
And remove the `Object` change revert to migrate prop as we destruct it here.